### PR TITLE
CI - fix Test Design System / Design System Tests workflow

### PR DESCRIPTION
**Pull request by KVNLS**
### ✅ Checklist

- [x] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [ ] **Impact of the changes:** 
  - ...

### 📝 Description

_Replace this text by a clear and concise description of what this pull request is about and why it is needed. Be sure to explain the problem you're addressing and the solution you're proposing._
_For libraries, you can add a code sample of how to use it._
_For bugfixes, you can explain the previous behavior and how it was fixed._
_In case of visual features, please attach screenshots or video recordings to demonstrate the changes._

### ❓ Context

- **JIRA or GitHub link**: 

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

### CODE REVIEW

**Review by KVNLS — COMMENTED** [review]
I can't approve as I'm the author of the PR but the fix is great! Well done!

File: .github/workflows/build-and-test-pr.yml
Comment (LucasWerey) [inline .github/workflows/build-and-test-pr.yml:71]: Shouldn't it be changed ?

Comment (themooneer) [reply .github/workflows/build-and-test-pr.yml:71]: yes ! before the merge

Comment (themooneer) [reply .github/workflows/build-and-test-pr.yml]: done ☑️

**Review by LucasWerey — APPROVED** [review]

**Pull request merged**
